src/libraries/System.Security.Cryptography/tests/wasm.helix.targets

```diff
@@ -0,0 +1,25 @@
+<Project>
+  <PropertyGroup>
+    <_CryptoProjectName>System.Security.Cryptography.Tests</_CryptoProjectName>
+    <System_Security_Cryptography_Tests_TargetName Condition="'$(Scenario))' == 'WasmTestOnBrowser'">System_Security_Cryptography_Tests_Target</System_Security_Cryptography_Tests_TargetName>
+  </PropertyGroup>
+
+  <Target Name="System_Security_Cryptography_Tests_Target">
+    <ItemGroup>
+      <HelixWorkItem Include="$(Scenario)-managed-$(_CryptoProjectName)">
+        <PayloadArchive>$(TestArchiveTestsDir)$(_CryptoProjectName).zip</PayloadArchive>
+        <Command>$(HelixCommand)</Command>
+        <Timeout>$(_workItemTimeout)</Timeout>
+      </HelixWorkItem>
+
+      <HelixWorkItem Include="$(Scenario)-subtlecrypto-System.Security.Cryptography.Tests">
+        <PayloadArchive>$(TestArchiveTestsDir)$(_CryptoProjectName).zip</PayloadArchive>
+        <Command>$(HelixCommand)</Command>
+        <Timeout>$(_workItemTimeout)</Timeout>
+
+        <PreCommands Condition="'$(OS)' == 'Windows_NT'">set &quot;WasmXHarnessArgs=--web-server-use-cors&quot;</PreCommands>
```

Comment:
Do we need to append to the current WasmXHarnessArgs? Will this either overwrite, or get overwritten?

Reply:
hm do you mean if the project itself is setting an *environment variable* with this name?
As for the *property*, that gets expanded, and written to the run script, so it won't be affected.

Reply:
how does the run script work? doesn't it set the environmnet variable? For example:

`
PATH=/workspaces/runtime/artifacts/bin/chromedriver/chromedriver_linux64:$PATH XHARNESS_ARGS="-v diag --browser-arg=--no-sandbox --browser-path /workspaces/runtime/artifacts/bin/chrome/chrome-linux/chrome" ./RunTests.sh
`

If it does this, doesn't that overwrite the "set"/"export" here?

Reply:
If you are running this locally, then these targets won't be used. In case of helix, such arguments would get expanded into the script, like:

```
$HARNESS_RUNNER wasm $XHARNESS_COMMAND
	--app=. --output-directory=$XHARNESS_OUT
	-s dotnet.js.symbols --symbol-patterns wasm-symbol-patterns.txt --symbolicator WasmSymbolicator.dll,Microsoft.WebAssembly.Internal.SymbolicatorWrapperForXHarness
	$XHARNESS_ARGS
	$WasmXHarnessArgs
	-- 
	$WasmXHarnessMonoArgs
	--run WasmTestRunner.dll System.Security.Cryptography.Tests.dll $WasmTestAppArgs -notrait category=IgnoreForCI -notrait category=OuterLoop -notrait category=failing $RSP_FILE
```

And we can set the environment variable like here. It should be safe to append anyway though. I'll add that.